src/main/java/org/jabref/logic/formatter/bibtexfields/NormalizeEmDashesIEEEFormatter.java

```diff
@@ -0,0 +1,31 @@
+package org.jabref.logic.formatter.bibtexfields;
+
+import org.jabref.logic.cleanup.Formatter;
+import org.jabref.logic.l10n.Localization;
+
+public class NormalizeEmDashesIEEEFormatter extends Formatter {
```

Comment:
Remove IEEE. Also from the description. The used character is a normal unicode character. Not special for IEEE.


Related formatter: https://github.com/JabRef/jabref/blob/main/src/main/java/org/jabref/logic/formatter/bibtexfields/NormalizeEnDashesFormatter.java